packages/common/src/api/OmSupplyApiContext.tsx

```diff
@@ -59,9 +59,17 @@ export const OmSupplyApiProvider: FC<ApiProviderProps> = ({
     setApi(createOmSupplyApi(url));
   }, [url]);
 
-  return (
-    <Provider value={{ api, client, setUrl, setHeader }}>{children}</Provider>
+  const val = useMemo(
+    () => ({
+      api,
+      client,
+      setUrl,
+      setHeader,
+    }),
+    [api, client, setUrl, setHeader]
   );
+
+  return <Provider value={val}>{children}</Provider>;
```

Comment:
Thought this might have been an issue when I was looking around for problems. It wasn't. Might be in the future, though, so kept it!